src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -348,6 +350,33 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
         auto pass_config = manager.get_pass_config();
         manager.set_per_pass_validation(false);
 
+        // Transformation of SDPA to VLSDPA for QWen2.x-VL,
+        // Note: this should be applied before TransposeFusion.
+        manager.register_pass<ov::pass::SDPAToVLSDPA>();
+        // Disable SDPAToVLSDPA if XMX architectures is unavaiable or IGC incompatiable.
+        pass_config->set_callback<ov::pass::SDPAToVLSDPA>(
+                [&](const_node_ptr &) -> bool {
+                    auto& engine = m_context->get_engine();
+                    const auto& info = engine.get_device_info();
+                    if (!(info.arch >= cldnn::gpu_arch::xe_lp)) {
+                        return true;
+                    }
+
+                    if (!config.get_use_cm()) {
+                        util::log_message("You may miss SDPAToVLSDPA optimization for QWenVL model,"
+                                    "as CM for usage is disabled. Enable it by setting environment variable OV_GPU_USE_CM=ON.");
+                        return true;
+                    }
+
+                    if (!check_cm_jit_support(engine, config)) {
+                        util::log_message("You may miss SDPAToVLSDPA optimization for QWenVL model,"
```

Comment:
@praasz , do we have any rules or agreements on when we can use these common-level logs?

Reply:
I thought about them before. "OPENVINO_INFO, OPENVINO_DEBUG" they are not a good option to my case, as "ENABLE_OPENVINO_DEBUG" is by default disabled.